### ISSUE TITLE
Gtk-WARNING **: Failed to get the GNOME session proxy...

### ISSUE THREAD

**Issue by lishoujun** [flatpak/flatpak#376]
lishoujun@qdfae:~$ flatpak run org.gnome.gedit 

*\* (gedit:3): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-KfklTdaHeC: Connection refused

(gedit:3): Gtk-WARNING **: Failed to get the GNOME session proxy: The name org.gnome.SessionManager is not owned

(gedit:3): Gtk-WARNING **: Failed to get the Xfce session proxy: The name org.xfce.SessionManager is not owned

(gedit:3): Gtk-WARNING **: Failed to get an inhibit portal proxy: The name org.freedesktop.portal.Desktop is not owned

(gedit:3): Gtk-WARNING **: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.

**Comment by lishoujun** [comment]
" flatpak run org.gnome.gedit" run in this way seems no difference with "gedit"

**Comment by alexlarsson** [comment]
The accessibility bus is: 
https://github.com/flatpak/flatpak/issues/79

The other things are just gedit in a sandbox not being allowed to talk to various things (i.e. issues with the way the gedit app was built) and the last one is about xdg-desktop-portal not being installed.

**Comment by lishoujun** [comment]
thanks
